tests/Unit/Extension/CoreTransform/Rpc/GenerateAccessorHandlerTest.php

```diff
@@ -4,46 +4,124 @@
 
 use Phpactor\Extension\Rpc\Handler;
 use Phpactor\CodeTransform\Domain\SourceCode;
+use Phpactor\Extension\Rpc\Response\InputCallbackResponse;
+use Phpactor\Extension\Rpc\Response\Input\ListInput;
 use Phpactor\Extension\Rpc\Response\UpdateFileSourceResponse;
 use Phpactor\CodeTransform\Domain\Refactor\GenerateAccessor;
 use Phpactor\Extension\CodeTransformExtra\Rpc\GenerateAccessorHandler;
 use Phpactor\Tests\Unit\Extension\Rpc\HandlerTestCase;
+use Phpactor\WorseReflection\ReflectorBuilder;
 
 class GenerateAccessorHandlerTest extends HandlerTestCase
 {
-    const SOURCE = '<?php echo "foo";';
+    const SOURCE = <<<'PHP'
+<?php
+
+class Dummy
+{
+    private $foo;
+    public $bar;
+}
+PHP;
     const PATH = '/path/to';
-    const OFFSET = 1234;
-    const CONSTANT_NAME = 'FOOBAR';
+    const FOO_NAME = 'foo';
+    const BAR_NAME = 'bar';
+    const PROPERTIES_CHOICES = [self::FOO_NAME => self::FOO_NAME, self::BAR_NAME => self::BAR_NAME];
+    const GENERATE_ACCESSOR_ACTION = 'generate_accessor';
+    const CURSOR_OFFSET = 57;
```

Comment:
Here I would generally use a data provider, rather than constants, as normally you may want to vary some of these values for each test, rather than gradually building up the class constants over the tests lifetime. This _could_ also reduce the amount of code, as there is only one test with many inputs?

Reply:
Don't remember exactly, just find a test with constants so I did it too to not have to rewrite everything.
But since then the test change enough so I've almost rewrite it already :D

Reply:
I just take a look at it and I would like to let it like that if it's OK with you.
This is a really simple test class, using data providers would actually make more code.
Since the handler mainly forwards the information to a third party there is not a lot that will ever change.
I will not have the time this evening anyway so let me know for the next time.

Reply:
Yeah no worries, it's more about being consistent if anything (and I sometimes end up turning tests similar to this into data providers over time anyway :)